dynamiq/nodes/agents/react.py

```diff
@@ -498,6 +458,100 @@ def _extract_final_answer_xml(self, llm_output: str) -> str:
         logger.info(f"Agent {self.name} - {self.id}: Final answer found: {final_answer['answer']}")
         return final_answer["answer"]
 
+    def generate_input_formats(self, tools: list[ToolMixin]) -> str:
+        input_formats = []
+        for tool in tools:
+            params = [
+                " ".join(param[:-1]) for param in tool.input_params if param[-1].get("is_accessible_to_agent", True)
+            ]
+            input_formats.append(f" - {tool.name}\n" f"\t* {"\n\t* ".join(params)}")
```

Comment:
It seems there are errors with quotes

Reply:
It is just implicit string concatenation but flattened to one line. Or you mean something else?

Reply:
@maksDev123 
```
    line 467
    input_formats.append(f" - {tool.name}\n" f"\t* {"\n\t* ".join(params)}")
                                                      ^
SyntaxError: unexpected character after line continuation character
```

It might be better not to flatten it to a single line.

Reply:
Update:
```
input_formats.append(
                f" - {tool.name}\n \t* {{\n\t* " + "\n\t* ".join(params) + "}}"
            )
 ```